tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "jsx": "react",
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
     "strict": true
-  }
+  },
 }
```

Comment:
Your prettier configuration seems to have conflicting designs. In the `.prettierrc.json` file that I have included the project, it should not add trailing commas. Make sure your the correct formatter is installed and set in your vscode. Install the prettier extension.